pytest_inmanta/test_parameter/integer_parameter.py

```diff
@@ -36,4 +36,4 @@ class IntegerTestParameter(TestParameter[int]):
     """
 
     def validate(self, raw_value: object) -> int:
-        return int(raw_value)
+        return int(str(raw_value))
```

Comment:
Purely out of curiosity, could you elaborate on this?

Reply:
Just mypy stuff, `int` doesn't take a value of type object as argument.  So I convert it to string first